### ISSUE TITLE
Option to reverse axis

### ISSUE THREAD

**Issue by thongsav-usgs** [USGS-R/gsplot#96]
EG: for depth plotted on Y-axis, you might want to have higher numbers plotted toward the bottom of a graph.

**Comment by jordansread** [comment]
at present:

``` R
gs <- gsplot() %>%
points(y=c(3,1,2), x=1:3, xlim=c(0,NA),ylim=c(10,0), col="blue", pch=18, legend.name="Points", xlab="Index")
gs
```

![image](https://cloud.githubusercontent.com/assets/2349007/8661603/5c2f741c-297e-11e5-9063-f4ae056ee11d.png)

satisfies this, but doesn't automatically calc the limits.

**Comment by ldecicco-USGS** [comment]
I think this is worth putting on the near-er term task list. Perhaps a back-burner task for @lindsaycarr

**Comment by lindsayplatt** [comment]
fixed by #214
